### PR TITLE
Added new items DB and corrected scripts (part 3)

### PR DESCRIPTION
**Pull request by zackdreaver**
Please check the script before merge.

### CODE REVIEW

File: db/re/item_combo_db.txt [hunk 1]
```diff
@@ -270,7 +275,6 @@
 24168:24169:24177,{ bonus bMatk,5; if(getequiprefinerycnt(EQI_SHADOW_WEAPON)+getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L)>=25) bonus bIgnoreMdefRace,RC_Formless; }
 24168:24169:24178,{ bonus bMatk,5; if(getequiprefinerycnt(EQI_SHADOW_WEAPON)+getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L)>=25) bonus bIgnoreMdefRace,RC_Undead; }
 24168:24169:24179,{ bonus bMatk,5; if(getequiprefinerycnt(EQI_SHADOW_WEAPON)+getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L)>=25) bonus bIgnoreMdefRace,RC_Plant; }
-28326:28327,{ bonus bInt,8; bonus bStr,8; }
```
Comment (dastgir) [inline db/re/item_combo_db.txt]: So, this removed?? Are you sure??

File: db/re/item_db.conf [hunk 2]
```diff
@@ -39438,6 +39479,20 @@ item_db: (
...
 	Weight: 10
 	Loc: 2
+	Script: <"
+		bonus2 bAddDamageClass,2085,30;
+		bonus2 bAddDamageClass,2090,30;
+		bonus2 bAddDamageClass,2144,30;
+		bonus2 bAddDamageClass,2145,30;
+		bonus2 bAddDamageClass,2083,30;
+		bonus2 bAddDamageClass,2088,30;
+		bonus2 bAddDamageClass,2084,30;
```
Comment (dastgir) [inline db/re/item_db.conf]: I think, we don't have AddDamageClass bonus
All Class bonus are not there, we deal with races.

File: db/re/item_combo_db.txt (hunk 1 above)
Comment (zackdreaver) [inline db/re/item_combo_db.txt]: its moved to upper line not removed

File: db/re/item_db.conf (hunk 2 above)
Comment (zackdreaver) [inline db/re/item_db.conf]: Well, i thought herc already have it,
i copied from item id 1558 o_O

Comment (dastgir) [inline db/re/item_db.conf]: Nvm this, seems like it was some other Class bonus...

File: db/re/item_combo_db.txt (hunk 1 above)
Comment (dastgir) [inline db/re/item_combo_db.txt]: I am blind :x

**Comment by zackdreaver** [comment]
Ok, i'm done with the commit,
its ready to merge

**Comment by dastgir** [comment]
Thanks For contribution

**Pull request merged**